app/helpers/sessions_helper.rb

```diff
@@ -4,15 +4,42 @@ def log_in user
   end
 
   def current_user
-    @current_user ||= User.find_by id: session[:user_id]
+    if (user_id = session[:user_id])
+      @current_user ||= User.find_by id: user_id
+    elsif (user_id = cookies.signed[:user_id])
+      user = User.find_by id: user_id
+      if user&.authenticated?(cookies[:remember_token])
+        log_in user
+        @current_user = user
+      end
+    end
   end
 
   def logged_in?
     current_user.present?
   end
 
+   # Logs out the current user.
   def log_out
+    forget(current_user)
     session.delete(:user_id)
     @current_user = nil
   end
+
+  def remember user
+    user.remember
+    cookies.permanent.signed[:user_id] = user.id
+    cookies.permanent[:remember_token] = user.remember_token
+  end
+
+  def authenticated? remember_token
```

Comment:
? 
this method shouldn't define here
please check again

Reply:
![Screenshot from 2021-09-10 16-51-24](https://user-images.githubusercontent.com/87750276/132827670-a8e1d86a-7c40-47e5-a091-954ce6d20bf9.png)

On the video it is define in the session_helper, I saw others people also same like me

Reply:
delete this function please
because it defined on user.rb model